static/src/javascripts/projects/commercial/modules/dfp/dfp-api.spec.js

```diff
@@ -0,0 +1,422 @@
+// @flow
+import $ from 'lib/$';
+import { noop } from 'lib/noop';
+import detect from 'lib/detect';
+import config from 'lib/config';
+
+import prepareGoogletag from 'commercial/modules/dfp/prepare-googletag';
+import getAdverts from 'commercial/modules/dfp/get-adverts';
+import getCreativeIDs from 'commercial/modules/dfp/get-creative-ids';
+import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
+import { commercialFeatures } from 'commercial/modules/commercial-features';
+
+jest.mock('common/modules/identity/api', () => ({
+    isUserLoggedIn: () => true,
+    getUserFromCookie: jest.fn(),
+}));
+jest.mock('common/modules/analytics/beacon', () => {});
+jest.mock('commercial/modules/dfp/refresh-on-resize', () => {});
+jest.mock('lib/detect', () => ({
```

Comment:
I haven't managed to get it to work, wandering if it's because that module's not converted yet